### ISSUE TITLE
What happened to weight percentile?

### ISSUE THREAD

**Issue by ckilbyfnp** [p2/growth-charts#30]
Weight percentile no longer being calculated by the app.

**Comment by DrPKrabman** [comment]
I really need this feature! Please provide it ASAP!
for me and I would imagine many doctors it is a basic functionality, with the current height, weight and percentiles dictated into each letter, and the difference between the percentiles for each an important visual.

**Comment by p2-apple** [comment]
Is this in the calculator? It seems to work for me. Does it show "n/a"? Then you need to select a different source by tapping top left.

**Comment by p2** [comment]
Is this in the calculator? It seems to work for me. Does it show "n/a"? Then you need to select a different source by tapping top left.

**Comment by DrPKrabman** [comment]
Thanks p2… it works with the CDCsource. That’s really useful. 
i would still request the weight percentiles for WHO source, as they’re the charts that I  usually use.

**Comment by p2** [comment]
You're right, interestingly WHO only works up to 10 years. It appears the app doesn't have data for kids older than that.

In any case, I'm not at Boston Children's anymore so I can't update. I do not know if they plan an update any time soon I'm afraid.

**Comment by p2** [comment]
(This is actually called out on https://p2.github.io/growth-charts/index/, under **Included Charts**)
